packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -311,7 +311,32 @@ export const actionCreators: ActionCreators = {
         return;
       }
       try {
-        await OAuthService.refreshTokenIfProjectExists(workspace);
+        try {
```

Comment:
@vinokurig Could we move it out from the previous try-catch block?

Could you add proper tests like this one https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/__tests__/actions.spec.ts#L405?

Reply:
> Could we move it out from the previous try-catch block?

No we can not, otherwise the workspace will not proceed to start if the `refreshTokenIfProjectExists` function fails.

Reply:
Sorry for confusion, fixed.